src/core/qgsvectorlayer.cpp

```diff
@@ -147,6 +147,7 @@ QgsVectorLayer::QgsVectorLayer( const QString &vectorLayerPath,
   , mSymbolFeatureCounted( false )
   , mEditCommandActive( false )
   , mReadExtentFromXml( readExtentFromXml )
+  , mIsRefreshOnNofifyEnabled( false )
```

Comment:
Can you move this initialization to the declaration in the header (c++11 style)?

Reply:
I don't mind, should I also move all other flags the same way ?

Reply:
Sure, if you feel like it! There's a clang tidy tool I was trying to use to do all this automatically, but it refused to work at all for me. Manually doing it is the next best thing ;)